Fix CI

The lack of ant-optional package causes CI error and oraclejdk8 item seems obsolete now.

```
BUILD FAILED
/home/travis/build/LTR4L/ltr4l/ltr4l-solr/build.xml:54: Problem: failed to create task or type junit
Cause: the class org.apache.tools.ant.taskdefs.optional.junit.JUnitTask was not found.
        This looks like one of Ant's optional components.
Action: Check that the appropriate optional JAR exists in
        -/usr/share/ant/lib
        -/home/travis/.ant/lib
        -a directory added on the command line with the -lib argument
Do not panic, this is a common problem.
The commonest cause is a missing JAR.
This is not a bug; it is a configuration problem
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,9 +1,9 @@
 language: java
 os: linux
 jdk:
-  - oraclejdk8
   - openjdk8
   - oraclejdk9
+  - openjdk9
 
 cache:
   directories:

--- a/.travis.yml
+++ b/.travis.yml
@@ -11,7 +11,7 @@ cache:
 
 before_install:
   - sudo apt-get update
-  - sudo apt-get install ivy -y
+  - sudo apt-get install ivy ant-optional -y
   - sudo ln -s -T /usr/share/java/ivy.jar /usr/share/ant/lib/ivy*.jar
 
 install:

